system/jlib/jlog.cpp

```diff
@@ -3239,15 +3239,18 @@ bool fetchLogByClass(LogQueryResultDetails & resultDetails, StringBuffer & retur
 //logAccessPluginConfig expected to contain connectivity and log mapping information
 typedef IRemoteLogAccess * (*newLogAccessPluginMethod_t_)(IPropertyTree & logAccessPluginConfig);
 
-IRemoteLogAccess &queryRemoteLogAccessor()
+IRemoteLogAccess *queryRemoteLogAccessor()
 {
-    return *logAccessor.query([]
+    return logAccessor.query([]
         {
-            Owned<IPropertyTree> logAccessPluginConfig = getGlobalConfigSP()->getPropTree("logAccess");
-#ifdef LOGACCESSDEBUG
-            if (!logAccessPluginConfig)
+            IRemoteLogAccess *remoteLogAccessor = nullptr;
```

Comment:
I think, as before, should keep a 'loading..' type message (that previously was in initWULogReader and Cws_logaccessEx::init), at the start of this function.
  PROGLOG("Loading remote log access plug-in.");